Makefile

```diff
@@ -0,0 +1,19 @@
+all:
+	mkdir -p build/
+	cd build/ && \
+		cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release ../ && \
```

Comment:
Building pistache is more nuanced than this.  Some users will want to enable SSL, or compile the examples or the unit tests even.  I field ~1 question per week that could be answered by the user on their own if they had examined the examples.